13_import.sql

```diff
@@ -359,14 +356,27 @@ CREATE TRIGGER after_insert_try_structure_update
 -- 8. new lets n - old lets 1 -> manual update needed
 -- 9. new lets n - old lets n -> manual update needed
 
-CREATE OR REPLACE FUNCTION qgep_import.manhole_quarantine_try_inlet_update() RETURNS trigger AS $BODY$
+CREATE OR REPLACE FUNCTION qgep_import.manhole_quarantine_try_let_update() RETURNS trigger AS $BODY$
   DECLARE 
+    is_inlet boolean;
+    let_kind text;
     new_lets integer;
     old_lets integer;
 BEGIN
+  is_inlet := TG_ARGV[0];
```

Comment:
might be a bit picky, but let's pass directly the string as argument, that would be clearer in the trigger code, i.e. 
`EXECUTE PROCEDURE qgep_import.manhole_quarantine_try_let_update( 'inlet' );` 
instead of 
`EXECUTE PROCEDURE qgep_import.manhole_quarantine_try_let_update( TRUE );`